readthedocs/projects/models.py

```diff
@@ -1055,10 +1055,19 @@ def get_git_service_class(self, fallback_to_clone_url=False):
         return service_cls
 
     @property
-    def git_provider_name(self):
-        """Get the provider name for project. e.g: GitHub, GitLab, Bitbucket."""
-        service_class = self.get_git_service_class(fallback_to_clone_url=True)
-        return service_class.allauth_provider.name if service_class else None
+    def is_github_project(self):
+        from readthedocs.oauth.services import GitHubService
+        return self.get_git_service_class(fallback_to_clone_url=True) == GitHubService
```

Comment:
This will be `in [GitHubService, GitHubAppService]` with https://github.com/readthedocs/readthedocs.org/pull/11942/